src/core/api/public_api.ts

```diff
@@ -1362,10 +1379,82 @@ class Player extends EventEmitter<IPublicAPIEvent> {
 
   /**
    * Update the playback rate of the video.
+   *
+   * This method's effect is persisted from content to content, and can be
+   * called even when no content is playing (it will still have an effect for
+   * the next contents).
+   *
+   * If you want to reverse effects provoked by `setPlaybackRate` before playing
+   * another content, you will have to call `setPlaybackRate` first with the
+   * default settings you want to set.
+   *
+   * As an example, to reset the speed to "normal" (x1) speed and to disable
+   * trickMode video tracks (which may have been enabled by a previous
+   * `setPlaybackRate` call), you can call:
+   * ```js
+   * player.setPlaybackRate(1, { preferTrickModeTracks: false });
+   * ```
+   *
+   * --
+   *
+   * This method can be used to switch to or exit from "trickMode" video tracks,
+   * which are tracks specifically defined to mimic the visual aspect of a VCR's
+   * fast forward/rewind feature, by only displaying a few video frames during
+   * playback.
+   *
+   * This behavior is configurable through the second argument, by adding a
+   * property named `preferTrickModeTracks` to that object.
+   *
+   * You can set that value to `true` to switch to trickMode video tracks when
+   * available, and set it to `false` when you want to disable that logic.
+   * Note that like any configuration given to `setPlaybackRate`, this setting
+   * is persisted through all future contents played by the player.
+   *
+   * If you want to stop enabling trickMode tracks, you will have to call
+   * `setPlaybackRate` again with `preferTrickModeTracks` set to `false`.
+   *
+   * You can know at any moment whether this behavior is enabled by calling
+   * the `areTrickModeTracksEnabled` method. This will only means that the
+   * RxPlayer will select in priority trickmode video tracks, not that the
+   * currently chosen video tracks is a trickmode track (for example, some
+   * contents may have no trickmode tracks available).
+   *
+   * If you want to know about the latter instead, you can call `getVideoTrack`
+   * and/or listen to `videoTrackChange` events. The track returned may have an
+   * `isTrickModeTrack` property set to `true`, indicating that it is a
+   * trickmode track.
+   *
+   * Note that switching to or getting out of a trickmode video track may
+   * lead to the player being a brief instant in a `"RELOADING"` state (notified
+   * through `playerStateChange` events and the `getPlayerState` method). When in
+   * that state, a black screen may be displayed and multiple RxPlayer APIs will
+   * not be usable.
```

Comment:
I like those long comments.

Joke aside, this is mainly because this is a method accessible though the API.

This can be useful in cases where an application developer use a language server that shows the documentation of called methods or when going to the definition.